documentation/tenant/assembly-managing-users.adoc

```diff
@@ -0,0 +1,27 @@
+:parent-context: {context}
+
+// The ID is used as an anchor for linking to the module. Avoid changing it after the module has been published to ensure existing links are not broken.
+[id='a-collection-of-modules']
+// If the assembly is reused in other assemblies in a guide, include {context} in the ID: [id='a-collection-of-modules-{context}'].
+= Managing users
+//If the assembly covers a task, start the title with a verb in the gerund form, such as Creating or Configuring.
+:context: managing-users
+// The `context` attribute enables module reuse. Every module's ID includes {context}, which ensures that the module has a unique ID even if it is reused multiple times in a guide.
+
+{ProductName} user management is supported when using the standard authentication service. Users can
+be managed using a kubernetes native REST API and custom resources using the command line tools.
+
+.Prerequisites
+
+* {ProductName} is installed with custom resources enabled
+* You have created an <<address_space>>
```

Comment:
Is <<address_space>> an xref? If so, better to use xref: syntax. 
Same with <<restapi-reference>> below